docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/config-based-source.md

```diff
@@ -105,51 +106,55 @@ Select this authentication option if the vendor API does not require any form of
   <div>
   Configure how the HTTP requests are created for your source.
 
-  :::danger protect your credentials
-  **Do NOT** include any sensitive information such as authentication secrets in this section. Use the authentication section for any sensitive information such as keys and passwords.
-  :::
+:::danger protect your credentials
+**Do NOT** include any sensitive information such as authentication secrets in this section. Use the authentication section for any sensitive information such as keys and passwords.
```

Comment:
We restrict the use of [bold text to UI elements](https://help.sumologic.com/docs/contributing/style-guide/#bold).

```suggestion
Do NOT include any sensitive information such as authentication secrets in this section. Use the authentication section for any sensitive information such as keys and passwords.
```